02_deployHono/ambassador_mappings.yaml

```
@@ -28,6 +39,7 @@ metadata:
   name: hono-kafka-external
 spec:
   port: 9092
+  #host: ${domain}
   service: hono-kafka-0-external:9092
```

```suggestion
  service: kafka-0-external:9092
```
This should fix the problem with resolving kafka endpoint when accessing trough <domain>:9092